blivet/events/handler.py

```diff
@@ -130,6 +131,7 @@ def _handle_add_event(self, event):
         if device is None and self._event_device_is_physical_disk(event):
             log.info("disk %s was added", udev.device_get_name(event.info))
             self.handle_device(event.info)
+            mpath_members.update_cache(udev.device_get_devname(event.info))
```

Comment:
Shouldn't we do this _before_ calling `handle_device`? (Doesn't it rely on the cache being up-to-date?)